src/test/groovy/com/jvm_bloggers/domain/command/top_posts_summary/GenerateTopPostsInMonthSummaryCommandHandlerIntegrationSpec.groovy

```diff
@@ -84,17 +88,20 @@ class GenerateTopPostsInMonthSummaryCommandHandlerIntegrationSpec extends Spring
         popularPosts.first().getPosition() == 1
         popularPosts.first().getNumberOfClicks() == 10
 
-        and: "each clicks count should be one less starting from 10"
+        and: 'each clicks count should be one less starting from 10'
         popularPosts.collect {it.getNumberOfClicks()} == [10L, 9L, 8L, 7L, 6L]
+
+        and: 'there was an event generated informing about successful summary generation'
+        !facebookPostRepository.findAll().isEmpty()
```

Comment:
Shouldn't the assertion be a little more specific?

Reply:
The only outcome is that one new entity is added to the repository. As the integration test is separate from each other there should be no other entities of this kind in the DB when executing test.